components/scream/tests/coupled/homme_physics/CMakeLists.txt

```diff
@@ -17,20 +17,24 @@ set (NEED_LIBS cld_fraction shoc p3 scream_rrtmgp rrtmgp ${NETCDF_C} ${dynLibNam
 # Add source files
 set (SRC shoc_cld_p3_rrtmgp.cpp ${SCREAM_BASE_DIR}/src/physics/rrtmgp/tests/rrtmgp_test_utils.cpp)
 
-# Test Homme standalone
+# Test Homme + SHOC + CLD + P3 + RRTGMP
+#if (NOT SCREAM_AUTOTESTER)
+#  CreateUnitTest(homme_physics_stand_alone "homme_physics_stand_alone.cpp" "${NEED_LIBS}" EXE_ARGS "< namelist.nl" LABELS "dynamics;driver;shoc;cld;p3;rrtmgp;physics")
```

Comment:
Can you remove the commented old version?

Reply:
oh duh... My bad